templates/electron-app/src/js/api/tasks.js

```diff
@@ -0,0 +1,36 @@
+import RequestWatcher from './request-watcher';
+
+let protocol = 'ws:';
+if (window.location.protocol === 'https:') {
+  protocol = 'wss:';
+}
+const host = 'localhost:8102'; 
```

Comment:
This changed to not try to detect the host the app is running on, because it needs to always talk to localhost since the server is running within Electron/node